### ISSUE TITLE
Fatal Error when trying to import .xml Dumps while Installation

### ISSUE THREAD

**Issue by BenniGotchi** [lansuite/lansuite#440]
## Expected Behavior

While Installation of Lansuite: Upload .xml dump of LansuiteData.
Import works flawless

## Current Behavior

Import fails with a Fatal error:
```
Fatal error: Uncaught Error: Call to a member function GetUploadFileType() on null in D:\MyProjects\git\lansuite\modules\install\wizard.php:239 Stack trace: #0 D:\MyProjects\git\lansuite\index_module.inc.php(55): include_once() #1 D:\MyProjects\git\lansuite\index.php(437): include_once('D:\\MyProjects\\g...') #2 {main} thrown in D:\MyProjects\git\lansuite\modules\install\wizard.php on line 239
```

## Possible Solution

Fix name inconsitency in 
https://github.com/lansuite/lansuite/blob/b75af3c28d17596cad868e2601bb32bb364a1e7a/modules/install/wizard.php#L7
https://github.com/lansuite/lansuite/blob/b75af3c28d17596cad868e2601bb32bb364a1e7a/modules/install/wizard.php#L239

## Steps to Reproduce (for bugs)

1. Perform a fresh installation with current Master
2. Select 'Import XML' and provide a Lansuite xml dump

## Context

Problem only present in the install wizard.
After a 'clean' installation, without importing old data, an afterwards performed import via the Admin Panel works.

## Your Environment

* Version used: current master
* Operating System and version: Windows 10 64bit, XAMPP, PHP7.3.2, MariaDB Version 10.1.38

**Comment by M4LuZ** [comment]
I guess this can be closed now with the merge of #441?

**Comment by BenniGotchi** [comment]
Yes.
How can that be handled in an automated way?
With #442 and #444 the issie was closed automatically with finishing the PR?

**Comment by M4LuZ** [comment]
> With #442 and #444 the issie was closed automatically with finishing the PR?

The statement `fixes #442` in the pull request does all the magic.
That automatically closes the referenced issue as the PR is supposed to fix it
Use `refs #442`  if you just want to create a reference in the related ticket

**Comment by BenniGotchi** [comment]
OK, thanks for the explanation 👍

Am Di., 9. Apr. 2019 um 14:33 Uhr schrieb M4LuZ <notifications@github.com>:
